src/yaml/yaml.cr

```diff
@@ -14,7 +15,33 @@ module YAML
   alias Type = String | Hash(Type, Type) | Array(Type) | Nil
   alias EventKind = LibYAML::EventType
 
-  def self.load(data)
+  # Deserializes a document from YAML format.
+  #
+  # ```yaml
```

Comment:
I don't think we have a YAML highlighter? Might be worth having the docs use highlightjs at some point.